lib/providers/hlr-lookups-provider.js

```diff
@@ -44,7 +61,7 @@ HLRLookupsProvider.prototype.hlrLookup = function hlrLookup(number) {
     }
 
     if (!isProviderLookupEnabled) {
-      logger.info(logFunc('Returning dummy lookup result because actual calls are prohibited (env ENABLE_HLR_LOOKUPS not set to true)'));
+      logWarningIfRequired(logger, logFunc);
       if (!dummyResponse) {
         dummyResponse = require('./dummy_data/hlr-lookups-provider.json').data;
```

Comment:
Hm I guess I could rewrite that. This is code from before so I did mockClient that does this for asyncHlrLookup but it is based on the ENABLE_HLR_LOOKUPS. I'll think how to do this here and maybe add the client as a dependency so it can be easily switched in test code and still not introduce breaking changes.